### DISCUSSION TITLE
Dash

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#279] [Ideas]
Please consider supporting Dash. Dash is a fast cryptocurrency common for payments and has been around for years.

**Comment by MrNaif2018** [comment]
If electrum for dash is supported (like electrum-ltc does) then it can work. I'll look into it once our CI issue is resolved

**Comment by skeutmeier** [comment]
Any news on this?

**Reply by MrNaif2018** [reply]
Hi, we would need electrum-dash to continue it's development. You can read up on what happened to it

**Comment by skeutmeier** [comment]
like this?
https://github.com/Bertrand256/electrum-dash/tree/4.1.7.0

Please let me know if you need a specific feature that is not implemented yet. This project was abandoned for about a year and is just now picked up again. If you are missing a feature, I will propose that to the dev-team

**Reply by MrNaif2018** [reply]
Oh, didn't know it. Is it officially maintained?

**Reply by skeutmeier** [reply]
As official as it gets in a DAO.

There is a development team called Dash Incubator, which is funded through Dash's funding mechanism. They officially picked up electrum just know after it was abandoned a year ago because of the russian invasion. 


And, yes, I agree. It was difficult to find. Even for me, and I consider myself part of the Dash Community
